### PR TITLE
Feat posts UI explore page

### PR DESCRIPTION
**Pull request by aishwarya-bm**
refactored code to create individual components for better page structure\
created UI for posts list and the comments list on the post

### CODE REVIEW

File: src/RoutePaths.jsx [hunk 1]
```diff
@@ -1,18 +1,54 @@
...
-          <Route path="/profile/:userId" element={<Profile />}></Route>
-          <Route path="/post/:postId" element={<Post />}></Route>
-          <Route path="/bookmarks" element={<Bookmarks />}></Route>
-          <Route path="/" element={<Login />}></Route>
+          <Route
+            path="/"
+            element={
+              <MainPage>
+                <Explore />
+              </MainPage>
```
Comment (nishasen) [inline src/RoutePaths.jsx]: you can use <outlet /> here

**Review by nishasen — APPROVED** [review]
Looks good added one suggestion

**Review by sadathshariff — APPROVED** [review]
Looks clean and good use of MUI Components

File: src/RoutePaths.jsx (hunk 1 above)
Comment (aishwarya-bm) [reply src/RoutePaths.jsx]: made changes to use Outlet from router

**Pull request merged**